frontend/src/Admin/actions.js

```diff
@@ -31,6 +40,19 @@ export const fetchShow = showId => (dispatch, getState, client) => {
     .catch(err => dispatch(displayError(err.message)))
 }
 
+
+export const fetchPortfolioPeriod = portfolioPeriodId => (dispatch, getState, client) => {
+  return client
+    .query({
+      query: PortfolioQuery,
+      variables: {
+        id: portfolioPeriodId
+      }
+    }) // TODO: Dispatch loading action & loading finished action
```

Comment:
These TODOs also echo the ones left by the last team for their redux actions. I wanted to leave them so that it's clear that the new operations would also be served by extra the loading actions.